tachyon/rs/base/container_util.h

```diff
@@ -45,6 +45,17 @@ absl::Span<R> ConvertRustSliceToCppSpan(rust::Slice<T> slice) {
   return {reinterpret_cast<R*>(slice.data()), slice.size()};
 }
 
+template <typename T>
+rust::Vec<T> CreateEmptyVector(size_t size) {
```

Comment:
How about changing the function name to `CreateDefaultVector` or `CreateInitializedVector `? That's why, the returned data might be mistakenly interpreted as a vector of length 0.